lib/node.js

```diff
@@ -56,31 +58,39 @@ export default class Node {
     ].join('');
   }
 
-  clone (overrides = { }) {
+  clone (overrides) {
+    overrides = overrides || {};
+
     let cloned = cloneNode(this);
 
-    for ( let name in overrides ) {
+    for (let name in overrides) {
       cloned[name] = overrides[name];
     }
 
     return cloned;
   }
 
-  cloneBefore (overrides = { }) {
+  cloneBefore (overrides) {
+    overrides = overrides || {};
+
     let cloned = this.clone(overrides);
 
     this.parent.insertBefore(this, cloned);
     return cloned;
   }
 
-  cloneAfter (overrides = { }) {
+  cloneAfter (overrides) {
+    overrides = overrides || {};
+
     let cloned = this.clone(overrides);
 
     this.parent.insertAfter(this, cloned);
     return cloned;
   }
 
-  replaceWith (...nodes) {
+  replaceWith () {
+    let nodes = Array.prototype.slice.call(arguments);
```

Comment:
`Array.from` should work here, right?

http://node.green/#Array-static-methods